### ISSUE TITLE
Shipment Notification Beautification

### ISSUE THREAD

**Issue by lauriewayne** [openfoodfoundation/wishlist#71]
## What is the need / problem?
The shipment notification generated by OFN lacks important and well-formatted information, which reduces the notification's functionality and reflects poorly on OFN and on the user/enterprise.

## Which type of users does this problem affect (and how many, if known)?

Shoppers_
Hub Managers (MCFEs)_
Producers_

## Potential solutions that will solve the problem  ?[[brainstorming to list feature candidates]
The email that is currently generated by the system looks like this:

![so ugly](https://user-images.githubusercontent.com/37803238/126510857-8f781fc3-b07e-4653-b895-ceb0fc28dfee.jpg)

A more readable and professional-looking email that is more on the level of order confirmations and invoices, might look like this (a designer with a better eye could do a better job here!):

![not as ugly](https://user-images.githubusercontent.com/37803238/126512806-9fab4a05-df9e-4c30-9c09-9cf7ffc1481c.jpg)

## Is Papercut or feature candidate?

_Add papercut or feature label to answer the question. If you are not sure, don't worry product team will cover :-)_

**Comment by jaycmb** [comment]
I d say more than Papercut
(related to MCFE Research > "I need to send alerts to my customers")

**Comment by RachL** [comment]
@lauriewayne yes this is papercut material, I will open a new issue and it will be able for you to select in the next round of papercut.

**Comment by lauriewayne** [comment]
Thank you @RachL in future I am happy to open the issue, I need the practice and your time is precious! :-)

**Comment by RachL** [comment]
Papercut: https://github.com/openfoodfoundation/wishlist/issues/128

**Comment by drummer83** [comment]
Just wondering if it made sense if for the list of products we used the same design in the shipment notification as we do in the order confirmation and order cancellation emails. Maybe even implementation could be copy paste?
Any arguments against this, @lauriewayne, @openfoodfoundation/train-drivers-product-owners ?
![Screenshot_20220527-000156_Gmail.jpg](https://user-images.githubusercontent.com/1790176/170587327-558433f0-f9ca-444e-a01b-4c385a1b158b.jpg)

**Comment by lauriewayne** [comment]
@drummer83 that sounds like a completely fabulous idea! Love it. Thank you!

**Comment by mgrigoriev8109** [comment]
If it's possible I'd love to tackle that and copy the order confirmation/order cancellation email design into the shipment notification email design.

**Comment by drummer83** [comment]
Sure @mgrigoriev8109! Since this is still a wishlist item without real specification, I would love to see a quick feedback from @openfoodfoundation/train-drivers-product-owners first to check if this is in line with their plans.
